### ISSUE TITLE
Error saving the composite key

### ISSUE THREAD

**Issue by dimxyp** [Angelelz/WinHelloUnlock#61]
Hi,
I am using this plugin for almost 2 years, but biometrics does not work in last days.
Remove and re-install plugin DLL gave me below message

```
---------------------------
KeePass
---------------------------
Error saving the composite key: The system cannot find the path specified.
Cannot add credential to Vault
---------------------------
OK   
---------------------------
```

The above message appearing after first install sequence and first wizard popup and Windows Security acceptation (finger or pin)
```
---------------------------
HelloUnlock
---------------------------
Do You want to set WinHelloUnlock for ........kdbx now?
---------------------------
Yes   No   
---------------------------
```

Also I cleared Plugin Data under Database Maintenance option, without luck.
I also tried some previous DLL versions of WinHelloUnlock, same behavior when click 'Enable WinHelloUnlock for this Database' (option menu) 

anyone with same issue? 

_Win10 21H1
KeePass 2.50(64bit) [installed plugins: KeePassRPC-1.14.0.0 / KeeTheme-0.9.4.0]_

**Comment by dimxyp** [comment]
Is it possible that we have conflicts using synced database on biometrics for multiple computers?

**Comment by Angelelz** [comment]
Looks like your issue is with the windows credential manager. Is your computer managed by an organization (company, school)? Did you update windows recently?

**Comment by dimxyp** [comment]
Yes, it is a corporate managed computer, and windows updates also managed by corporate SCCM, any workaround?
